From b8d770528351a017da822b0985de62986198fcdf Mon Sep 17 00:00:00 2001
From: Ole Streicher <olebole@debian.org>
Date: Mon, 1 Apr 2024 00:15:05 +0200
Subject: [PATCH] Allow empty IRAFARCH in irafpath()

---
 unix/os/irafpath.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/unix/os/irafpath.c b/unix/os/irafpath.c
index 06da15616..55b087e4b 100644
--- a/unix/os/irafpath.c
+++ b/unix/os/irafpath.c
@@ -71,7 +71,8 @@ irafpath (char *fname)
 	 */
 	strcpy (pathname, (char *)hostdir);
 	strcat (pathname, "bin");
-	if ( (irafarch = getenv("IRAFARCH")) ) {
+	if ( (irafarch = getenv("IRAFARCH"))
+	     && (strlen(irafarch) > 0) ) {
 	    strcat (pathname, ".");
 	    strcat (pathname, irafarch);
 	}
@@ -90,7 +91,8 @@ irafpath (char *fname)
 	/* Try BIN - use IRAFARCH if defined. */
 	strcpy (pathname, (char *)irafdir);
 	strcat (pathname, "bin");
-	if ( (irafarch = getenv("IRAFARCH")) ) {
+	if ( (irafarch = getenv("IRAFARCH"))
+	     && (strlen(irafarch) > 0) ) {
 	    strcat (pathname, ".");
 	    strcat (pathname, irafarch);
 	}
-- 
2.43.0

