From: Ole Streicher <olebole@debian.org>
Date: Sat, 15 Jul 2023 19:55:41 +0200
Subject: fix DESTDIR in Makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3786734..40529ff 100644
--- a/Makefile
+++ b/Makefile
@@ -180,7 +180,7 @@ prefix ?= /usr/local
 install: noao/bin/x_quad.e
 	mkdir -p $(DESTDIR)$(prefix)/lib/iraf $(DESTDIR)$(prefix)/bin $(DESTDIR)$(prefix)/share/man/man1 $(DESTDIR)/etc/iraf
 	cp -a -f bin* dev extern include lib local math mkpkg noao pkg sys test unix \
-	         $(prefix)/lib/iraf
+	         $(DESTDIR)$(prefix)/lib/iraf
 	$(MAKE) config binary_links strip iraf=$(prefix)/lib/iraf/ bindir=$(prefix)/bin
 	cp -f $(hlib)mkiraf.man $(DESTDIR)$(prefix)/share/man/man1/mkiraf.1
 	cp -f $(hlib)ecl.man $(DESTDIR)$(prefix)/share/man/man1/ecl.1
