From 188e45d953a60164a1b22512191678de933e7729 Mon Sep 17 00:00:00 2001
From: Ole Streicher <olebole@debian.org>
Date: Fri, 15 Mar 2024 15:44:54 +0100
Subject: [PATCH] Force setting of local terminfo database

---
 xgterm/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xgterm/main.c b/xgterm/main.c
index 2ec1437..2817d8c 100644
--- a/xgterm/main.c
+++ b/xgterm/main.c
@@ -1026,6 +1026,8 @@ char **argv;
 #ifdef I18N
         setlocale(LC_ALL, NULL);
 #endif
+	if (getenv("TERMINFO") == NULL)
+	  putenv("TERMINFO=/usr/local/share/terminfo");
 
 	ProgramName = argv[0];
 
-- 
2.43.0

